test/fix.js

```diff
@@ -92,7 +92,7 @@ for (const [name, testFix] of Object.entries(testMethods)) {
           'package.json': pkg({ name: '@npmcli/Test-Package' }),
         }
         await t.rejects(
-          testFix(t, testdir, { strict: true }),
+          testFix(t, testdir, { strict: true, allowLegacyCase: false }),
```

Comment:
Why did these (allowLegacyCase) change in the tests?

Reply:
Yep, they didn't need to, it was when were were still talking about how these were going to pass through vs inline. I've reverted these 2